.github/workflows/java8_integration_tests_ft.yml

```diff
@@ -53,6 +53,8 @@ jobs:
           ALLUXIO_DOCKER_FORK_COUNT=1 \
           ALLUXIO_DOCKER_NO_TTY=true \
           ALLUXIO_DOCKER_GIT_CLEAN=true \
+          ALLUXIO_DOCKER_MVN_PROJECT_LIST_COMPILE=dora/tests \
+          ALLUXIO_DOCKER_MVN_PROJECT_LIST_TEST=dora/tests \
```

Comment:
It seems wired if I set the COMPILE to dora/tests. Actually no one test is run its corresponding workflow log. So what modules should we set here? @Xenorith

Reply:
from a recent 2.x PR https://github.com/Alluxio/alluxio/actions/runs/6766062935/job/18386591178, i see this mvn command run:
`mvn -Duser.home=/home/jenkins test -Pdeveloper -Dmaven.main.skip -Dskip.protoc=true -Dmaven.javadoc.skip -Dlicense.skip=true -Dcheckstyle.skip=true -Dfindbugs.skip=true -Dsurefire.forkCount=1 -fn -DfailIfNoTests=false --fail-at-end '"-Dtest=alluxio.server.ft.journal.raft.**,alluxio.server.ft.journal.ufs.**"' -pl tests`
which ran these tests:
```
2023-11-06T03:54:35.2986689Z 03:54:35.297 [INFO] -------------------------------------------------------
2023-11-06T03:54:35.2987667Z 03:54:35.297 [INFO]  T E S T S
2023-11-06T03:54:35.3064138Z 03:54:35.297 [INFO] -------------------------------------------------------
2023-11-06T03:54:35.8643715Z 03:54:35.846 [INFO] Running alluxio.server.ft.journal.raft.EmbeddedJournalIntegrationTestFaultTolerance
2023-11-06T03:57:40.5298131Z 03:57:40.528 [INFO] Tests run: 8, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 184.67 s - in alluxio.server.ft.journal.raft.EmbeddedJournalIntegrationTestFaultTolerance
2023-11-06T03:57:41.3294493Z 03:57:41.321 [INFO] Running alluxio.server.ft.journal.raft.EmbeddedJournalIntegrationTestResizing
2023-11-06T03:58:39.6630498Z 03:58:39.662 [INFO] Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 58.334 s - in alluxio.server.ft.journal.raft.EmbeddedJournalIntegrationTestResizing
2023-11-06T03:58:40.3712567Z 03:58:40.370 [INFO] Running alluxio.server.ft.journal.raft.EmbeddedJournalIntegrationTestTransferLeadership
2023-11-06T04:02:46.2645885Z 04:02:46.263 [INFO] Tests run: 8, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 245.887 s - in alluxio.server.ft.journal.raft.EmbeddedJournalIntegrationTestTransferLeadership
2023-11-06T04:02:47.0629348Z 04:02:47.053 [INFO] Running alluxio.server.ft.journal.ufs.RenameFailureJournalTest
2023-11-06T04:03:02.8038131Z 04:03:02.803 [INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 15.746 s - in alluxio.server.ft.journal.ufs.RenameFailureJournalTest
2023-11-06T04:03:03.4837581Z 04:03:03.483 [INFO] Running alluxio.server.ft.journal.ufs.SecondaryMasterTest
2023-11-06T04:03:07.3471604Z 04:03:07.345 [INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 3.858 s - in alluxio.server.ft.journal.ufs.SecondaryMasterTest
2023-11-06T04:03:08.0465610Z 04:03:08.038 [INFO] Running alluxio.server.ft.journal.ufs.UfsConfigurationJournalTest
2023-11-06T04:03:13.0289545Z 04:03:13.024 [INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 4.981 s - in alluxio.server.ft.journal.ufs.UfsConfigurationJournalTest
2023-11-06T04:03:13.7273654Z 04:03:13.723 [INFO] Running alluxio.server.ft.journal.ufs.UfsJournalIntegrationTest
2023-11-06T04:03:37.7712308Z 04:03:37.770 [INFO] Tests run: 15, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 24.042 s - in alluxio.server.ft.journal.ufs.UfsJournalIntegrationTest
```

whereas in this PR, the command is
`mvn -Duser.home=/home/jenkins test -Dmaven.main.skip -Dskip.protoc=true -Dmaven.javadoc.skip -Dlicense.skip=true -Dcheckstyle.skip=true -Dfindbugs.skip=true -Dsort.skip -Dsurefire.forkCount=1 -am -pl dora/tests -fn -DfailIfNoTests=false -Dsurefire.failIfNoSpecifiedTests=false --fail-at-end '-Dtest=alluxio.server.ft.journal.raft.**,alluxio.server.ft.journal.ufs.**'`

so i see two differences:
- the correct one has `-pl` at the end instead of in the middle
- the correct one does not have `-am`

Reply:
Is `-Pdeveloper` not related?

Reply:
Sorry, I found profile `developer` is deleted now.